daffodil-test/src/test/resources/org/apache/daffodil/extensions/type_calc/inputTypeCalc.tdml

```diff
@@ -183,23 +180,6 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
-  <tdml:parserTestCase name="RepType_lengthKind_inherited_01"
-    root="keysetValue_03" model="inputTypeCalc-Embedded.dfdl.xsd"
-    description="Verify that the lengthKind property is inherited from the repType, not the element using the repType">
-
-    <tdml:document>
-    <tdml:documentPart type="byte">
-    01
-    </tdml:documentPart>
-    </tdml:document>
-    <tdml:infoset>
-      <tdml:dfdlInfoset xmlns:xs="http://www.w3.org/2001/XMLSchema"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-        <keysetValue_03>one</keysetValue_03>
-      </tdml:dfdlInfoset>
-    </tdml:infoset>
-  </tdml:parserTestCase>
-
```

Comment:
Are these tests faulty or do they just not trigger the particular issue? The schema seems correct to me and might be worth keeping if it's testing something different than other tests.

Reply:
Is there value in keeping the tests? Do they test anything different? I assume at point they were broken and have since been fixed (maybe not with your recent change though?) so maybe worth keeping?

Reply:
In the end I don't think it was testing anything different than the previously existing keyset value tests.  So the test I removed is unnecessary.